src/defs_version.h

```diff
@@ -49,5 +49,5 @@
 //
 #ifndef VERSION
     // In that case, this value is used
-    #define VERSION "2.1-alpha-pre"
```

Comment:
There was already a not above saying that windows version should be MAJOR.MINOR.patch

so I made a more visible note.

What do you think about enforcing the versionning scheme in SConscript ?